config/server.go

```diff
@@ -29,6 +34,13 @@ type serverGRPC struct {
 	Port string
 }
 
+func serverConfigLoader() {
+	viper.SetDefault("SERVER_DEDUP_ENABLED", "true")
```

Comment:
Can we make false as the default since its a low latency ingestion system and the expectation for de-dupe is generally post-ingestion problem to solve.

Reply:
done.